test/karma/behaviorSpec.js

```diff
@@ -0,0 +1,522 @@
+}
+
+describe('A Torso Behavior', function() {
+
+  setupInjectionSite.apply(this);
+
+  it('exists', function() {
+    expect(TorsoBehavior).toBeDefined();
+  });
+
+  it('can be instantiated', function() {
+    expect(new TorsoBehavior()).toBeDefined();
+  });
+
+  it('can be extended', function() {
+    expect(TorsoBehavior.extend).toBeDefined();
+  });
+
+  describe('instance', function() {
+    beforeEach(function() {
+      this.behavior = new TorsoBehavior();
+    });
+
+    it('can set and get properties', function() {
+      this.behavior.set('property', 'value');
+      var property = this.behavior.get('property');
+      expect(property).toBe('value');
+    });
+
+  });
+
+  describe('when instantiated', function() {
+    it('options are passed to initialize', function() {
+      var BehaviorWithInitialize = TorsoBehavior.extend({
+        initialize: function() {
+          this.initializeArguments = arguments;
+        }
+      });
+      var options = {propertyKey: 'propertyValue'};
+
+      var behaviorWithInitialize = new BehaviorWithInitialize(options);
+      expect(behaviorWithInitialize.initializeArguments).toBeDefined();
+      expect(behaviorWithInitialize.initializeArguments[0]).toEqual(options);
+    });
+
+    // Don't override constructor in normal usage of Behavior
+    it('arguments are passed to constructor', function() {
+      var BehaviorWithConstructor = TorsoBehavior.extend({
+        constructor: function() {
+          this.constructorArguments = arguments;
+          TorsoBehavior.apply(this, arguments);
+        }
+      });
+      var options = {propertyKey: 'propertyValue'};
+      var view = new TorsoView();
+
+      var behaviorWithConstructor = new BehaviorWithConstructor(options, view);
+      expect(behaviorWithConstructor.constructorArguments[0]).toBe(options);
+      expect(behaviorWithConstructor.constructorArguments[1]).toBe(view);
+    });
+
+    it('will set reference to view', function() {
+      var view = new TorsoView();
+      var behaviorCreatedWithView = new TorsoBehavior({view: view});
+      expect(behaviorCreatedWithView.view).toBeDefined();
+      expect(behaviorCreatedWithView.view).toEqual(view);
+    });
+  });
+
+  it('A view can reference its behavior using an alias', function() {
+    var viewWithBehavior = new ViewWithBehavior();
+    var behaviorInView = viewWithBehavior.getBehavior('torsoBehavior');
+    expect(behaviorInView).toBeDefined();
+  });
+
+  it('Behaviors are not static fields, i.e. they are instantiated per view instance', function() {
+    var firstViewWithBehavior = new ViewWithBehavior();
+    var secondViewWithBehavior = new ViewWithBehavior();
+
+    var firstBehavior = firstViewWithBehavior.getBehavior('torsoBehavior');
+    var secondBehavior = secondViewWithBehavior.getBehavior('torsoBehavior');
+
+    expect(firstBehavior).not.toBe(secondBehavior);
+  });
+
+  it('A view can have multiple behaviors distinguished by alias', function() {
```

Comment:
May wish to add specifications around when the lifecycle methods of multiple behaviors run.